app/controllers/comments_controller.rb

```diff
@@ -1,17 +1,21 @@
 class CommentsController < ApplicationController
   def create
-    return head :forbidden unless params["imahuman"].to_s == ENV["IMAHUMAN"].to_s
+    return head :forbidden unless validate_humanity(params)
 
     if project.comments.create(comment_params.merge(user: current_user))
       notify_project
-      redirect_to project_path(project), notice: "Comment added!"
+      redirect_to project_path(project), notice: 'Comment added!'
     else
-      redirect_to project_path(project), error: "Oops, we couldn't add that comment!"
+      redirect_to project_path(project), error: 'Oops, we couldn\'t add that comment!'
     end
   end
 
   private
 
+  def validate_humanity(params)
+    params['imahuman'].to_s == ENV['IMAHUMAN'].to_s && verify_recaptcha(params)
```

Comment:
Is it possible to just strip this code out? ReCAPTCHA would be authoritative. I don't see much problem in keeping both, although one isn't working now.